src/Microsoft.ServiceFabric.Actors/Runtime/ActorConcurrencyLock.cs

```diff
@@ -93,37 +93,37 @@ internal int Test_CurrentCount
             await this.reentrantLock.WaitAsync(cancellationToken);
             try
             {
-                // A new logical call context is appended to every outgoing method call.
+                // Specifies a new logical call context is appended to every outgoing method call.
                 // The received callContext is of form callContext1callContext2callContext3... 
                 // thus to check if incoming call was made from the current calls in progress
-                // we need to check if the incoming call context starts with the currentCallContext
+                // we need to check if the incoming call context starts with the currentCallContext.
                 var startsWith = incomingCallContext.StartsWith(this.currentCallContext);
 
                 if (startsWith)
                 {
-                    // the incoming call is part of the current call chain
+                    // Indicates the incoming call is a part of the current call chain.
 
-                    // the messaging layer may deliver duplicate messages, therefore if the 
+                    // The messaging layer may deliver duplicate messages, therefore if the 
                     // incomingCallContext is same as currentCallContext it is a duplicate message
                     // this is because every outgoing call from actors has a new callContext appended
-                    // to the currentCallContext
+                    // to the currentCallContext.
                     if (incomingCallContext.Length == this.currentCallContext.Length)
                     {
                         throw new DuplicateMessageException(string.Format(CultureInfo.CurrentCulture,
                                                             SR.ErrorDuplicateMessage,this.GetType()));
                     }
 
                     //
-                    // this is a reentrant call
+                    // This is a reentrant call.
                     //
 
-                    // if the reentrancy is disallowed, throw and exception
+                    // Disallows actors to be reentrant. In this case if an actor sends a reentrant message to another actor an exception will be thrown.
```

Comment:
This is a code comment telling the follow of the code, this change is not needed.